templates/Web/_composition/Flask/Page.Flask.AddSampleData.ForAllExceptList/README_postaction.md

```diff
@@ -0,0 +1,25 @@
+﻿## Next Steps
+
+//{[{
+
+### Sample Data
+
+Replace the sample data stored in /server/sampleData.py.
+Replace the default images stored in /src/images.
+//}]}
+
```

Comment:
Under what case should I see this Sample Data portion?

I just generated a flask without cosmos, I did not see this part in the README

Reply:
It shows when I generates a flask with Cosmos.

Reply:
You would see Sample Data if you select grid/masterDetail page irrespective of if you have chosen cosmosDB or not.
If you do not select masterDetail page and grid page (only select List page), you would see sample Data only if cosmos is not selected.

Reply:
Did you select any pages when you generated a Flask application without Cosmos?

Reply:
I selected Grid and List page